scripts/set-system-env.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if uname -s | grep -q Darwin; then
+  SED=gsed
+else
+  SED=sed
+fi
+
+PROTOCOL="http"
+IP=$(ipconfig getifaddr en0)
```

Comment:
Fails on non-macOS and macOS with a non-default network adapter.

Better add comments for this, allowing users to fill in by hand.

Reply:
Added in new commits.